website/docs/r/mongo_cluster.html.markdown

```diff
@@ -0,0 +1,129 @@
+---
+subcategory: "MongoCluster"
+layout: "azurerm"
+page_title: "Azure Resource Manager: `azurerm_mongo_cluster"
+description: |-
+  Manages an Azure Cosmos DB for MongoDB vCore.
+---
+
+# azurerm_mongo_cluster
+
+Manages an Azure Cosmos DB for MongoDB vCore.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "East US"
+}
+
+resource "azurerm_mongo_cluster" "example" {
+  name                   = "example-mc"
+  resource_group_name    = azurerm_resource_group.example.name
+  location               = azurerm_resource_group.example.location
+  administrator_username = "adminTerraform"
+  administrator_password = "QAZwsx123"
+  shard_count            = "1"
+  compute_tier           = "Free"
+  high_availability_mode = "Disabled"
+  storage_size_in_gb     = "32"
+}
+
+```
+
+## Example Usage (Preview feature GeoReplicas)
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "East US"
+}
+
+resource "azurerm_mongo_cluster" "example" {
+  name                   = "example-mc"
+  resource_group_name    = azurerm_resource_group.example.name
+  location               = azurerm_resource_group.example.location
+  administrator_username = "adminTerraform"
+  administrator_password = "QAZwsx123"
+  shard_count            = "1"
+  compute_tier           = "M30"
+  high_availability_mode = "ZoneRedundantPreferred"
+  storage_size_in_gb     = "64"
+  preview_features       = ["GeoReplicas"]
+}
+
+resource "azurerm_mongo_cluster" "example_geo_replica" {
+  name                = "example-mc-geo"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = "Central US"
+  source_server_id    = azurerm_mongo_cluster.example.id
+  source_location     = azurerm_mongo_cluster.example.location
+  create_mode         = "GeoReplica"
+
+  lifecycle {
+    ignore_changes = ["administrator_username", "high_availability_mode", "preview_features", "shard_count", "storage_size_in_gb", "compute_tier", "version"]
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for the Azure Cosmos DB for MongoDB vCore. Changing this forces a new resource to be created.
```

Comment:
Can you please update all references in these docs from `Azure Cosmos DB for MongoDB vCore` to 
```suggestion
* `name` - (Required) The name which should be used for the MongoDB Cluster. Changing this forces a new resource to be created.
```

Reply:
Fixed.